### PR TITLE
add words to make Windows cmd.exe environment variable expansion syntax clearer

### LINKED ISSUE #3065: Update dynamic runtime asset path docs
### Description

The Windows execution environment uses different envvar expansion syntax compared to other agents. We hardcode the shell environments for command execution. This is not currently documented.

See https://github.com/sensu/sensu-go/issues/4184.

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [x] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
https://docs.sensu.io/sensu-go/latest/plugins/assets/#dynamic-runtime-asset-path

### Additional context
The existing docs content was added in https://github.com/sensu/sensu-docs/pull/2494.

Discussion at https://sensu.slack.com/archives/C60EEQFH8/p1617822874033400

### PR DESCRIPTION
**Pull request by jspaleta**
## Description
Clarification on environment variable expansion syntax for Windows agents.

## Motivation and Context
Windows agents use different execution shell and require different envvar syntax.
Want to be clear when examples are for Windows agents or for POSIX agents.

### CODE REVIEW

**Comment by hillaryfraley** [comment]
Thanks again @jspaleta 

I made a couple copy editing changes. If they are all 👍 I can propagate this to all docs versions and publish. I also want to be sure I got the reference 44 in the right spot (for the % symbol reference, right?)

**Comment by hillaryfraley** [comment]
@jspaleta I added the execution environment info to the agent and backend references as well as the sensuctl index page. Wdyt? I didn't list Windows on the backend page since Windows is agent and sensuctl only, but I'm not sure this is correct.

File: content/sensu-go/6.1/plugins/assets.md [hunk 1]
```diff
@@ -445,7 +452,7 @@ If you want to access a dynamic runtime asset path in a handler or mutator comma
 
 For example, you can reference the dynamic runtime asset [`sensu-plugins-windows`][4] from your check or hook resources using either the environment variable or the `assetPath` function:
 
-- `$SENSU_PLUGINS_WINDOWS_PATH/include/config.yaml`
+- `$%SENSU_PLUGINS_WINDOWS_PATH%/include/config.yaml`
```
Comment (jspaleta) [inline content/sensu-go/6.1/plugins/assets.md]: need to drop the `$` character here for windows cmd.exe envvar substitution .

Comment (hillaryfraley) [inline content/sensu-go/6.1/plugins/assets.md]:
```suggestion
- `%SENSU_PLUGINS_WINDOWS_PATH%/include/config.yaml`
```

**Pull request merged**